autoload/airline/extensions/lsp.vim

```diff
@@ -0,0 +1,63 @@
+" MIT License. Copyright (c) 2013-2019 François-Xavier Carton et al.
+" Plugin: https://github.com/prabirshrestha/vim-lsp
+" vim: et ts=2 sts=2 sw=2
+
```

Comment:
I thought it would be better if such a specification was at the beginning of the file.
What do you think? @chrisbra @fx-carton 

```vim
if !get(g:, 'lsp_loaded', 0)
  finish
endif
```

Reply:
This is the same as the vim-ale extension. I don't know what is the rationale for not putting the check at the beginning. Maybe it's because airline might be loaded before vim-lsp?